helm/README.md

```diff
@@ -54,6 +54,21 @@ $ kubectl exec -it "$POD_NAME" -- /opt/code-marketplace add https://github.com/V
 In the future it will be possible to use Artifactory for storing and retrieving
 extensions instead of a persistent volume.
 
+## Adding custom certificate authorities
+
+If the location for retrieving extensions (or if using Artifactory storage) is not signed by a common CA, then create a secret in the deployed namespace:
+```
+kubectl create secret -n $namespace generic all-cas --from-file="certificate1.pem"=/path/to/certificate1.pem \
```

Comment:
Thinking it might look nice if all the `--from-file` are on their own line.

```suggestion
kubectl create secret -n $namespace generic all-cas \
 --from-file="certificate1.pem"=/path/to/certificate1.pem \
```